app/javascript/app/components/vulnerability-graph/vulnerability-graph-component.jsx

```diff
@@ -65,7 +65,7 @@ class VulnerabilityGraph extends PureComponent {
               {sectionData.data.rank.absolute}
               <span className={styles.countriesNumber}>
                 {' '}
-                of {countriesCount}
```

Comment:
As part of this task: https://www.pivotaltracker.com/story/show/152389076 , and the fact that the ranking is not over the total number of countries I'm now obtaining this value from the API. This means that the "countriesCount" is not needed for this component, as far as I can tell. So maybe someone that knows Javascript could come here and remove where needed.